common/src/main/scala/akka/contrib/persistence/mongodb/MongoReadJournal.scala

```diff
@@ -22,25 +25,50 @@ class MongoReadJournal(system: ExtendedActorSystem, config: Config) extends Read
   override def javadslReadJournal(): javadsl.ReadJournal = new JavaDslMongoReadJournal(new ScalaDslMongoReadJournal(impl))
 }
 
-class ScalaDslMongoReadJournal(impl: MongoPersistenceReadJournallingApi) extends scaladsl.ReadJournal with CurrentPersistenceIdsQuery with CurrentEventsByPersistenceIdQuery {
+class ScalaDslMongoReadJournal(impl: MongoPersistenceReadJournallingApi)
+    extends scaladsl.ReadJournal
+    with CurrentPersistenceIdsQuery
+    with CurrentEventsByPersistenceIdQuery
+    with EventsByPersistenceIdQuery{
 
-  def allEvents(): Source[EventEnvelope,Unit] =
-    Source.actorPublisher[EventEnvelope](impl.allEvents)
+  def currentAllEvents(): Source[EventEnvelope,Unit] =
+    Source.actorPublisher[EventEnvelope](impl.currentAllEvents)
```

Comment:
Can you update this one as well to use the `EventEnvelopeConverter` for consistency?  Definitely like that approach as far as isolating concerns (like the `offset`).